test/document_viewer_web/ensure_document_viewer_group_test.exs

```diff
@@ -4,20 +4,21 @@ defmodule DocumentViewerWeb.EnsureDocumentViewerGroupTest do
   alias DocumentViewerWeb.EnsureDocumentViewerGroup
 
   describe "init/1" do
-    test "passes options through unchanged" do
-      assert EnsureDocumentViewerGroup.init([]) == []
+    test "passes options through with the addition of the cognito group" do
+      assert EnsureDocumentViewerGroup.init([]) == [document_viewer_group: "document-viewer"]
     end
   end
 
   describe "call/2" do
     @tag :authenticated
     test "does nothing when user is in the document-viewer group", %{conn: conn} do
-      assert conn == EnsureDocumentViewerGroup.call(conn, [])
+      assert conn ==
+               EnsureDocumentViewerGroup.call(conn, document_viewer_group: "document-viewer")
```

Comment:
you can get the default arguments here (rather than hard-coding) by calling `EnsureDocumentViewerGroup.init([])`.